### PR TITLE
Updated the tensorflow API call

### PR DESCRIPTION
**Pull request by andimarafioti**
Newer versions of tensor flow don't accept calls to sparse_softmax_cross_entropy_with_logits if the logits and labels are not explicited.

### CODE REVIEW

**Comment by cocuh** [comment]
thx👍
